vendor/composer/installed.json

```diff
@@ -65,6 +65,6 @@
             "install-path": "../coingate/coingate-php"
         }
     ],
-    "dev": true,
+    "dev": false,
```

Comment:
why is this not dev ?

Reply:
Dev libraries for development purpose to check WP coding standards, so do not need to commit them. 

`"require": {
        "coingate/coingate-php": "^4.1"
    },
    "require-dev": {
        "10up/wp_mock": "dev-trunk",
        "dealerdirect/phpcodesniffer-composer-installer": "^0.7",
        "phpcompatibility/php-compatibility": "^9.1",
        "wp-coding-standards/wpcs": "^2.0"
    }`